silkrpc/commands/trace_api.cpp

```diff
@@ -121,12 +121,77 @@ boost::asio::awaitable<void> TraceRpcApi::handle_trace_call_many(const nlohmann:
 
 // https://eth.wiki/json-rpc/API#trace_rawtransaction
 boost::asio::awaitable<void> TraceRpcApi::handle_trace_raw_transaction(const nlohmann::json& request, nlohmann::json& reply) {
+    auto params = request["params"];
+    if (params.size() < 2) {
+        auto error_msg = "invalid trace_rawTransaction params: " + params.dump();
```

Comment:
`error_msg` can be made `const` here and below

Reply:
done